lib/crossing.rb

```diff
@@ -2,10 +2,18 @@
 
 # Documentation incoming
 class Crossing
-  def initialize(s3_client)
+  def initialize(s3_client, s3_client_unenc=nil)
     raise CrossingMisconfigurationException if s3_client.nil?
     raise CrossingMisconfigurationException unless s3_client.is_a? Aws::S3::Encryption::Client
     @s3_client = s3_client
```

Comment:
Maybe rename this to `s3_client_encrypted` to avoid confusion?

Reply:
Good idea, done